src/sentry/static/sentry/app/views/organizationMonitors/monitorForm.jsx

```diff
@@ -16,6 +17,19 @@ import withOrganization from 'app/utils/withOrganization';
 
 import MonitorModel from './monitorModel';
 
+const SCHEDULE_TYPES = [['crontab', 'Crontab'], ['interval', 'Interval']];
+
+const MONITOR_TYPES = [['cron_job', 'Cron Job']];
+
+const INTERVALS = [
+  ['minute', 'minute(s)'],
+  ['hour', 'hour(s)'],
+  ['day', 'day(s)'],
+  ['week', 'week(s)'],
```

Comment:
maybe we should rethink these and just only expose minutes from a backend POV - it'd certainly make schedule configuration easier